packages/web/components/Dashboard/MyFeed/MyFeed.tsx

```diff
@@ -115,130 +68,43 @@ const MyFeed: React.FC<Props> = ({ currentUser, initialSearchFilters }) => {
     router.push({ ...router, query: newQuery })
   }
 
-  const onSearchChange = useCallback((val): void => setSearchState(val), [])
-
-  const onTopicAdd = useCallback(
-    (id: number): void => {
-      setSelectedTopicsFilters([...selectedTopicsFilters, id])
-      resetPagination()
-    },
-    [selectedTopicsFilters],
-  )
-
-  const onTopicRemove = useCallback(
-    (id: number): void => {
-      setSelectedTopicsFilters(selectedTopicsFilters.filter((topicId) => topicId !== id))
-      resetPagination()
-    },
-    [selectedTopicsFilters],
-  )
-
-  const onLanguageAdd = useCallback(
-    (id: number): void => {
-      setSelectedLanguageFilters([...selectedLanguageFilters, id])
-      resetPagination()
-    },
-    [selectedLanguageFilters],
-  )
-
-  const onLanguageRemove = useCallback(
-    (id: number): void => {
-      setSelectedLanguageFilters(selectedLanguageFilters.filter((languageId) => languageId !== id))
-      resetPagination()
-    },
-    [selectedLanguageFilters],
-  )
-
   useEffect(() => {
     const searchFilters = {
-      languages: selectedLanguageFilters,
-      topics: selectedTopicsFilters,
-      needsFeedback: needsFeedbackFilter,
-      hasInteracted: hasInteractedFilter,
+      languages: postQueryVars?.languages,
```

Comment:
Do we need the optional chain here? I think these should be non-nullable

Reply:
Good call, removed it